sdk/storage/storage-blob/test/node/sas.spec.ts

```diff
@@ -467,6 +468,10 @@ describe("Shared Access Signature (SAS) generation Node.js only", () => {
       }
     ]);
 
+    if (env.TEST_MODE && (env.TEST_MODE.startsWith("live") || env.TEST_MODE.startsWith("record"))) {
+      await delay(30 * 1000);
+    }
```

Comment:
Instead of this, just import the `delay` exposed from the recorder.
```ts
import {delay} from "@azure/test-utils-recorder";
await delay(<milliseconds>);
```
This `delay` has no effect if the `TEST_MODE` is `"playback"`.
`delay` works as expected(`await delay(<milliseconds>)`) if the `TEST_MODE` is "live"/"record".

Reply:
[GUIDELINES.md#importing-delay-from-recorder](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/GUIDELINES.md#importing-delay-from-azuretest-utils-recorder)